Dockerfile.x86

```diff
@@ -35,6 +35,13 @@ copy openshift_identity.py /app/openshift-acct-mgt/openshift_identity.py
 copy openshift_rolebindings.py /app/openshift-acct-mgt/openshift_rolebindings.py
 copy openshift_project.py /app/openshift-acct-mgt/openshift_project.py
 copy openshift_role.py /app/openshift-acct-mgt/openshift_role.py
+copy openshift_resource_quota.py /app/openshift-acct-mgt/openshift_resource_quota.py
```

Comment:
Two comments:

First, It helps to be consistent in your capitalization. If you're going to use `RUN` and `COPY` elsewhere, use `COPY` here rather than `copy`.

Secondly, this series of individual `COPY` statements is inefficient. Reorganize your source directory if necessary, then use a single `COPY` statement to copy your local `openshift-acct-mgt` directory to `/app/openshift-acct-mgt`.